keyboards/lendunistus/rpneko65/rev1/config.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2023 lendunistus
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
```

Comment:
This should really exist here instead of the `<keyboard>.h`
```suggestion

#define CAPS_LOCK_ENABLE true
```